docs/reference/analysis/analyzers/whitespace-analyzer.asciidoc

```diff
@@ -133,7 +133,8 @@ PUT /whitespace_example
       "analyzer": {
         "rebuilt_whitespace": {
           "tokenizer": "whitespace",
-          "filter": [         <1>
+          "filter": [
+            "lowercase"       <1>
```

Comment:
I believe the `lowercase` token filter was included with the [standard analyzer](https://www.elastic.co/guide/en/elasticsearch/reference/master/analysis-standard-analyzer.html#_definition_4) example because it's included with the built-in definition.

That's not the case with the `whitespace` analyzer. The only thing included in its built-in definition is the `whitespace` tokenizer.

As-is, these changes would invalidate the previous sentence:

> This would recreate the built-in `whitespace` analyzer and you can use it as a starting point for further customization: